### ISSUE TITLE
openjdk8_j9_extended.functional_x86-64_windows Shared cache offset out of bounds Segmentation error vmState=0x0005ff04

### ISSUE THREAD

**Issue by JasonFengJ9** [eclipse/openj9#8175]
Failure link
------------

```
03:45:18  openjdk version "1.8.0_242-internal"
03:45:18  OpenJDK Runtime Environment (build 1.8.0_242-internal-jenkins_2019_12_22_18_25-b00)
03:45:18  Eclipse OpenJ9 VM (build ibm_sdk-be96c7eec3, JRE 1.8.0 Windows Server 2016 amd64-64-Bit Compressed References 20191222_265 (JIT enabled, AOT enabled)
03:45:18  OpenJ9   - be96c7eec3
03:45:18  OMR      - 614d68a5b
03:45:18  JCL      - b2a1abf9261 based on jdk8u242-b04)
```

Optional info
-------------

Failure output (captured from console output)
---------------------------------------------

```
03:53:43  Buildfile: C:\Users\jenkins\workspace\Test_openjdk8_j9_extended.functional_x86-64_windows_Nightly\openjdk-tests\TKG\scripts\build_test.xml
03:53:43  Assertion failed at C:\Users\jenkins\workspace\Build_JDK8_x86-64_windows_Nightly\build\windows-x86_64-normal-server-release\vm\compiler\env\J9SharedCache.cpp:457: false
03:53:43  VMState: 0x0005ff04
03:53:43  	Shared cache offset out of bounds
03:53:43  compiling org/apache/tools/ant/property/ParseProperties$$Lambda$8/0000000000000000.apply(Ljava/lang/Object;)Ljava/lang/Object; at level: cold
03:53:43  
03:53:43  Unhandled exception
03:53:43  Type=Segmentation error vmState=0x0005ff04
03:53:43  Windows_ExceptionCode=c0000005 J9Generic_Signal=00000004 ExceptionAddress=0000000055A9FEB6 ContextFlags=0010001f
03:53:43  Handler1=000000005579BB80 Handler2=00007FF9D210CCF0 InaccessibleWriteAddress=0000000000000000
03:53:43  RDI=000000000F15B038 RSI=000000000F15B030 RAX=0000000000000000 RBX=0054626FED9460B8
03:53:43  RCX=BA89F29BF1BF0000 RDX=0000000000000000 R8=0000000000000069 R9=00000000000000CB
03:53:43  R10=00000000000002C0 R11=0000000000000059 R12=0000000000000003 R13=FFFFFFFFF0EA4FC8
03:53:43  R14=000000000E78BBE0 R15=000000000EA25150
03:53:43  RIP=0000000055A9FEB6 RSP=000000000F15AE90 RBP=000000000F15B060 GS=002B
03:53:43  FS=0053 ES=002B DS=002B
03:53:43  XMM0 0000000000000000 (f: 0.000000, d: 0.000000e+000)
03:53:43  XMM1 0000000000000000 (f: 0.000000, d: 0.000000e+000)
03:53:43  XMM2 0000000000000000 (f: 0.000000, d: 0.000000e+000)
03:53:43  XMM3 0000000000000000 (f: 0.000000, d: 0.000000e+000)
03:53:43  XMM4 0000000000000000 (f: 0.000000, d: 0.000000e+000)
03:53:43  XMM5 0000000000000000 (f: 0.000000, d: 0.000000e+000)
03:53:43  XMM6 0000000000000000 (f: 0.000000, d: 0.000000e+000)
03:53:43  XMM7 0000000000000000 (f: 0.000000, d: 0.000000e+000)
03:53:43  XMM8 0000000000000000 (f: 0.000000, d: 0.000000e+000)
03:53:43  XMM9 0000000000000000 (f: 0.000000, d: 0.000000e+000)
03:53:43  XMM10 0000000000000000 (f: 0.000000, d: 0.000000e+000)
03:53:43  XMM11 0000000000000000 (f: 0.000000, d: 0.000000e+000)
03:53:43  XMM12 0000000000000000 (f: 0.000000, d: 0.000000e+000)
03:53:43  XMM13 0000000000000000 (f: 0.000000, d: 0.000000e+000)
03:53:43  XMM14 0000000000000000 (f: 0.000000, d: 0.000000e+000)
03:53:43  XMM15 0000000000000000 (f: 0.000000, d: 0.000000e+000)
03:53:43  Module=C:\Users\jenkins\workspace\Test_openjdk8_j9_extended.functional_x86-64_windows_Nightly\openjdkbinary\j2sdk-image\jre\bin\compressedrefs\j9jit29.dll
03:53:43  Module_base_address=0000000055750000 Offset_in_DLL=000000000034feb6
03:53:43  
03:53:43  Method_being_compiled=org/apache/tools/ant/property/ParseProperties$$Lambda$8/0000000000000000.apply(Ljava/lang/Object;)Ljava/lang/Object;
03:53:43  Target=2_90_20191222_265 (Windows Server 2016 10.0 build 14393)
03:53:43  CPU=amd64 (4 logical CPUs) (0x1fff8d000 RAM)
03:53:43  ----------- Stack Backtrace -----------
03:53:43  Java_java_lang_invoke_MutableCallSite_invalidate+0x2bd3f6 (0x0000000055A9FEB6 [j9jit29+0x34feb6])
03:53:43  Java_java_lang_invoke_MutableCallSite_invalidate+0x2bd66d (0x0000000055AA012D [j9jit29+0x35012d])
03:53:43  j9jit_testarossa+0x7e84 (0x00000000557C82E4 [j9jit29+0x782e4])
03:53:43  Java_java_lang_invoke_MutableCallSite_invalidate+0x9d998 (0x0000000055880458 [j9jit29+0x130458])
03:53:43  Java_java_lang_invoke_MutableCallSite_invalidate+0xa1638 (0x00000000558840F8 [j9jit29+0x1340f8])
03:53:43  Java_java_lang_invoke_MutableCallSite_invalidate+0x9b64d (0x000000005587E10D [j9jit29+0x12e10d])
03:53:43  Java_java_lang_invoke_MutableCallSite_invalidate+0x9fda2 (0x0000000055882862 [j9jit29+0x132862])
03:53:43  Java_java_lang_invoke_MutableCallSite_invalidate+0x9e77e (0x000000005588123E [j9jit29+0x13123e])
03:53:43  Java_java_lang_invoke_MutableCallSite_invalidate+0xa72e2 (0x0000000055889DA2 [j9jit29+0x139da2])
03:53:43  (0x000000005577F07A [j9jit29+0x2f07a])
03:53:43  Java_java_lang_invoke_MutableCallSite_invalidate+0x1127fd (0x00000000558F52BD [j9jit29+0x1a52bd])
03:53:43  Java_java_lang_invoke_MutableCallSite_invalidate+0x108bd5 (0x00000000558EB695 [j9jit29+0x19b695])
03:53:43  Java_java_lang_invoke_MutableCallSite_invalidate+0x245289 (0x0000000055A27D49 [j9jit29+0x2d7d49])
03:53:43  (0x000000005576EA21 [j9jit29+0x1ea21])
03:53:43  Java_java_lang_invoke_MutableCallSite_invalidate+0x25095c (0x0000000055A3341C [j9jit29+0x2e341c])
03:53:43  Java_java_lang_invoke_MutableCallSite_invalidate+0x24f44b (0x0000000055A31F0B [j9jit29+0x2e1f0b])
03:53:43  Java_java_lang_invoke_MutableCallSite_invalidate+0x24c5c0 (0x0000000055A2F080 [j9jit29+0x2df080])
03:53:43  Java_java_lang_invoke_MutableCallSite_invalidate+0x260809 (0x0000000055A432C9 [j9jit29+0x2f32c9])
03:53:43  (0x00000000557A6E76 [j9jit29+0x56e76])
03:53:43  (0x00000000557A89FB [j9jit29+0x589fb])
03:53:43  j9port_init_library+0x1b4f6 (0x00007FF9D210DEC6 [j9prt29+0x1dec6])
03:53:43  j9port_init_library+0x1b6f4 (0x00007FF9D210E0C4 [j9prt29+0x1e0c4])
03:53:43  (0x00000000557A929C [j9jit29+0x5929c])
03:53:43  (0x00000000557A9862 [j9jit29+0x59862])
03:53:43  (0x00000000557A45CA [j9jit29+0x545ca])
03:53:43  (0x00000000557A757E [j9jit29+0x5757e])
03:53:43  j9port_init_library+0x1b72f (0x00007FF9D210E0FF [j9prt29+0x1e0ff])
03:53:43  (0x00000000557A8E4D [j9jit29+0x58e4d])
03:53:43  omrthread_detach+0x1fd (0x00007FF9D7B25D6D [J9THR29+0x5d6d])
03:53:43  _endthreadex+0x43 (0x00000000561D1D9F [msvcr100+0x21d9f])
03:53:43  _endthreadex+0xdf (0x00000000561D1E3B [msvcr100+0x21e3b])
03:53:43  BaseThreadInitThunk+0x14 (0x00007FF9DFE284D4 [KERNEL32+0x84d4])
03:53:43  RtlUserThreadStart+0x21 (0x00007FF9E24AE851 [ntdll+0x6e851])
03:53:43  ---------------------------------------
03:53:43  JVMDUMP039I Processing dump event "gpf", detail "" at 2019/12/23 03:52:49 - please wait.
03:53:43  JVMDUMP032I JVM requested System dump using 'C:\Users\jenkins\workspace\Test_openjdk8_j9_extended.functional_x86-64_windows_Nightly\openjdk-tests\TKG\core.20191223.035249.20776.0001.dmp' in response to an event
03:53:47  JVMDUMP010I System dump written to C:\Users\jenkins\workspace\Test_openjdk8_j9_extended.functional_x86-64_windows_Nightly\openjdk-tests\TKG\core.20191223.035249.20776.0001.dmp
03:53:47  JVMDUMP032I JVM requested Java dump using 'C:\Users\jenkins\workspace\Test_openjdk8_j9_extended.functional_x86-64_windows_Nightly\openjdk-tests\TKG\javacore.20191223.035249.20776.0002.txt' in response to an event
03:53:47  JVMDUMP010I Java dump written to C:\Users\jenkins\workspace\Test_openjdk8_j9_extended.functional_x86-64_windows_Nightly\openjdk-tests\TKG\javacore.20191223.035249.20776.0002.txt
03:53:47  JVMDUMP032I JVM requested Snap dump using 'C:\Users\jenkins\workspace\Test_openjdk8_j9_extended.functional_x86-64_windows_Nightly\openjdk-tests\TKG\Snap.20191223.035249.20776.0003.trc' in response to an event
03:53:47  JVMDUMP010I Snap dump written to C:\Users\jenkins\workspace\Test_openjdk8_j9_extended.functional_x86-64_windows_Nightly\openjdk-tests\TKG\Snap.20191223.035249.20776.0003.trc
03:53:47  JVMDUMP007I JVM Requesting JIT dump using 'C:\Users\jenkins\workspace\Test_openjdk8_j9_extended.functional_x86-64_windows_Nightly\openjdk-tests\TKG\jitdump.20191223.035249.20776.0004.dmp'
```

**Comment by pshipton** [comment]
Similar to #7890

**Comment by pshipton** [comment]
This was resolved by #8679
